[2024-11-02T13:29:51.441+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-02-20T00:00:00+00:00 [queued]>
[2024-11-02T13:29:51.452+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-02-20T00:00:00+00:00 [queued]>
[2024-11-02T13:29:51.452+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-02T13:29:51.472+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotSchemaSubmitOperator): submit_schemas> on 2024-02-20 00:00:00+00:00
[2024-11-02T13:29:51.477+0000] {standard_task_runner.py:57} INFO - Started process 3211 to run task
[2024-11-02T13:29:51.480+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'schema_dag', 'submit_schemas', 'scheduled__2024-02-20T00:00:00+00:00', '--job-id', '1238', '--raw', '--subdir', 'DAGS_FOLDER/schema_dag.py', '--cfg-path', '/tmp/tmp3k0rz1y8']
[2024-11-02T13:29:51.484+0000] {standard_task_runner.py:85} INFO - Job 1238: Subtask submit_schemas
[2024-11-02T13:29:51.541+0000] {task_command.py:415} INFO - Running <TaskInstance: schema_dag.submit_schemas scheduled__2024-02-20T00:00:00+00:00 [running]> on host 5e812a6a7c02
[2024-11-02T13:29:51.634+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='minhtranquang' AIRFLOW_CTX_DAG_ID='schema_dag' AIRFLOW_CTX_TASK_ID='submit_schemas' AIRFLOW_CTX_EXECUTION_DATE='2024-02-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-20T00:00:00+00:00'
[2024-11-02T13:29:51.635+0000] {pinot_schema_operator.py:33} ERROR - An error occured: 'function' object has no attribute 'glob'
[2024-11-02T13:29:51.645+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=schema_dag, task_id=submit_schemas, execution_date=20240220T000000, start_date=20241102T132951, end_date=20241102T132951
[2024-11-02T13:29:51.693+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-02T13:29:51.715+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
